packages/framework/src/Model/Product/Filter/PriceRangeRepository.php

```diff
@@ -83,6 +84,9 @@ protected function getPriceRangeByProductsQueryBuilder(QueryBuilder $productsQue
         $priceRangeData = $queryBuilder->getQuery()->execute();
         $priceRangeDataRow = reset($priceRangeData);
 
-        return new PriceRange($priceRangeDataRow['minimalPrice'], $priceRangeDataRow['maximalPrice']);
+        return new PriceRange(
+            Money::fromValue($priceRangeDataRow['minimalPrice'] ?? 0),
```

Comment:
you can you `fromString()` in my opinion because in the query, there are used aggregation functions, so the Doctrine have no idea how to map these results. I think it is right that Doctrine returns string in this case (not our Money).

Reply:
It might be that case. Thanks for mentioning that, I was trying to postpone the decision here (same as in https://github.com/shopsys/shopsys/pull/838#discussion_r258407036).